crates/uv/src/commands/pip_compile.rs

```diff
@@ -369,7 +370,15 @@ pub(crate) async fn pip_compile(
         writeln!(
             writer,
             "{}",
-            format!("#    {}", cmd(include_index_url, include_find_links,)).green()
+            format!(
+                "#    {}",
+                cmd(
+                    include_index_url,
+                    include_find_links,
+                    custom_compile_command
+                )
```

Comment:
Alternatively, if you prefer to have it inline:
```suggestion
                custom_compile_command.unwrap_or(cmd(include_index_url, include_find_links))
```
and remove the changes in `cmd` function